dotCMS/src/main/java/com/dotmarketing/portlets/rules/parameter/type/DataType.java

```diff
@@ -23,6 +29,12 @@ public DataType(String id, String errorMessageKey) {
         this.errorMessageKey = errorMessageKey;
     }
 
+    public DataType(String id, String errorMessageKey, Map<String, TypeConstraint> restrictions) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'restrictions' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4925354)